src/accelgyrotemp_ros_publisher.cpp

```diff
@@ -50,6 +50,9 @@ void AccelGyroTempRosPublisher::run()
     RCLCPP_INFO(rclcpp::get_logger("rclcpp_accelgyrotemp"), "startThread: '%d'", this_id);
 
     bool success = false;
+    bool firstData = true;
+    bool activate = false;
```

Comment:
I see no reason to use this extra variable in the code success can be used, and this variable can be removed
please apply the same logic to the other publisher